Wikipedia/Code/NSDateFormatter+WMFExtensions.h

```diff
@@ -39,6 +39,6 @@
 
 + (instancetype)wmf_mediumDateFormatterWithoutTime;
 
-+ (instancetype)wmf_hyphenatedYearMonthDayFormatter;
++ (instancetype)wmf_englishHyphenatedYearMonthDayFormatter;
```

Comment:
not sure what you mean, this date formatter is meant to convert dates into english format (e.g. `2015-12-8`) regardless of the current device language.